demo/interhand3d_img_demo.py

```diff
@@ -0,0 +1,246 @@
+import os
+import os.path as osp
+from argparse import ArgumentParser
+
+import mmcv
+import numpy as np
+from xtcocotools.coco import COCO
+
+from mmpose.apis import inference_interhand_3d_model, vis_3d_pose_result
+from mmpose.apis.inference import init_pose_model
+from mmpose.core import SimpleCamera
+
+
+def _transfer_interhand_camera_param(interhand_camera_param):
+    """Transform the camera parameters in interhand2.6m dataset to the format
+    of SimpleCamera.
+
+    Args:
+        interhand_camera_param (dict): camera parameters including:
+            - camrot: 3x3, camera rotation matrix (world-to-camera)
+            - campos: 3x1, camera location in world space
+            - focal: 2x1, camera focal length
+            - princpt: 2x1, camera center
+
+    Returns:
+        param (dict): camera parameters including:
+            - R: 3x3, camera rotation matrix (camera-to-world)
+            - T: 3x1, camera translation (camera-to-world)
+            - f: 2x1, camera focal length
+            - c: 2x1, camera center
+    """
+    camera_param = {}
+    camera_param['R'] = np.array(interhand_camera_param['camrot']).T
+    camera_param['T'] = np.array(interhand_camera_param['campos'])[:,
+                                                                   np.newaxis]
+    camera_param['f'] = np.array(interhand_camera_param['focal'])[:,
+                                                                  np.newaxis]
+    camera_param['c'] = np.array(interhand_camera_param['princpt'])[:,
+                                                                    np.newaxis]
+    return camera_param
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument('pose_config', help='Config file for pose network')
+    parser.add_argument('pose_checkpoint', help='Checkpoint file')
+    parser.add_argument('--img-root', type=str, default='', help='Image root')
+    parser.add_argument(
+        '--json-file',
+        type=str,
+        default='',
+        help='Json file containing image info.')
+    parser.add_argument(
+        '--camera-param-file',
+        type=str,
+        default=None,
+        help='Camera parameter file for converting 3D pose predictions from '
+        ' the pixel space to camera space. If None, keypoints in pixel space'
+        'will be visualized')
+    parser.add_argument(
+        '--gt-joints-file',
```

Comment:
Please make sure that the ground truth is optional and the demo script can run without it.

Reply:
Yes, this demo can be used without gt_joints and camera parameters.

Reply:
Ok. It's better to add a description to explain that this is optional and what would happen if it is not provided.

Reply:
OK, I have added some comments.